src/frontend/screens/MapScreen/MapLayers/ObservationMapLayer.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import MapboxGL from '@rnmapbox/maps';
 
-import {useObservations} from '../../hooks/server/observations';
-import {usePresetsQuery} from '../../hooks/server/presets';
-import {useNavigationFromHomeTabs} from '../../hooks/useNavigationWithTypes';
+import {useObservations} from '../../../hooks/server/observations.ts';
+import {usePresetsQuery} from '../../../hooks/server/presets.ts';
+import {useNavigationFromHomeTabs} from '../../../hooks/useNavigationWithTypes.ts';
```

Comment:
out of curiosity - are you adding the file extensions manually or is that some external formatting feature being applied? Not opposed to it, but does add a little extra noise to the diff

Reply:
i think it is vscode doing that. I moved these files into a folder and vscode updated the imports. I will remove